components/global/product_switcher/product_switcher.tsx

```diff
@@ -60,6 +66,14 @@ const MenuItem = styled(Link)`
         text-decoration: none;
         color: inherit;
     }
+
+    i {
```

Comment:
Is this the proper way to style a compass component? Shouldn't we be doing something like:

```js
const StyledIcon = styled(Icon)`
    color: var(--sidebar-bg);
`
```